### ISSUE TITLE
undefined method checking empty hiera files

### ISSUE THREAD

**Issue by johnl** [mschuchard/puppet-check#1]
I've got a hiera yaml file that is empty (i.e: no yaml in it). puppet-check borks parsing it:
```
vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/lib/puppet-check/data_parser.rb:90:in `hiera': undefined method `each' for false:FalseClass (NoMethodError)
	from vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/lib/puppet-check/data_parser.rb:16:in `block in yaml'
	from vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/lib/puppet-check/data_parser.rb:9:in `each'
	from vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/lib/puppet-check/data_parser.rb:9:in `yaml'
	from vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/lib/puppet-check.rb:70:in `execute_parsers'
	from vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/lib/puppet-check.rb:32:in `run'
	from vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/lib/puppet-check/cli.rb:12:in `run'
	from vendor/bundle/ruby/2.3.0/gems/puppet-check-1.0.0/bin/puppet-check:5:in `<top (required)>'
	from vendor/bundle/ruby/2.3.0/bin/puppet-check:23:in `load'
	from vendor/bundle/ruby/2.3.0/bin/puppet-check:23:in `<main>'
```

**Comment by mschuchard** [comment]
Thank you for finding a bug. I will fix this in the next commit.

**Comment by mschuchard** [comment]
Fixed in a13dc9b4adc7c92e93a94cd5d2fb04b2ad5b1f01. This will be in the next release.

**Comment by johnl** [comment]
thank you for fixing it!
